cmd/root/auth.go

```diff
@@ -173,19 +174,23 @@ func SetWorkspaceClient(ctx context.Context, w *databricks.WorkspaceClient) cont
 	return context.WithValue(ctx, &workspaceClient, w)
 }
 
+var ErrNoConfiguration = errors.New("no configuration file found")
+
 func transformLoadError(path string, err error) error {
-	if os.IsNotExist(err) {
-		return fmt.Errorf("no configuration file found at %s; please create one first", path)
+	if os.IsNotExist(err) || errors.Is(err, fs.ErrNotExist) {
+		// downstreams need to handle these errors properly, but we erase the fs.ErrNotExist
+		// TODO: expose this as error through SDK
```

Comment:
Which paths are downstream of this, only/mostly the profile prompters?

Reply:
only prompters, see #914

Reply:
OK, so given that these are exported in this PR, thus are made into public APIs, can you please move the `transformLoadError` function as well as the constant error to the `databrickscfg` package? Only the two prompt functions below call into it after calling `databrickscfg.LoadProfiles`.

Then instead of:
```go
if err != nil && errors.Is(err, root.ErrNoConfiguration)
```

The check becomes more self explanatory:
```go
if err != nil && errors.Is(err, databrickscfg.ErrNoConfiguration)
```

Reply:
look, i can make this change, but it's not me who put this function into a wrong module ;) https://github.com/databricks/cli/pull/454/files

Reply:
You're turning it into an exported API here, that makes it different.